packages/vats/src/bootstrap.js

```diff
@@ -678,6 +672,13 @@ export function buildRootObject(vatPowers, vatParameters) {
         devices.bridge && makeBridgeManager(E, D, devices.bridge);
       const {
         ROLE,
+        // TODO: Don't make client bootstrap dependent on having just zero or
+        // one chains.  Instead, supply all the connections as input to the
+        // bootstrap (or have other ways of initializing per-connection
+        // bootstrap code).  Also use an abstract name for the connection
+        // instead of GCI so that a given chain can be followed across a GCI
+        // change such as in a hard-fork.
```

Comment:
interesting, that's an extra level of indirection I hadn't considered.. I'll try to keep that in mind when thinking about the configuration of network links